prepare-build.sh

```diff
@@ -150,26 +133,15 @@ BBLAYERS = " \
   ${SRCDIR}/meta-openembedded/meta-networking \
   ${SRCDIR}/meta-smartphone/meta-android \
   ${SRCDIR}/meta-openembedded/meta-python \
-  ${SRCDIR}/meta-openembedded/meta-filesystems \
-  ${SRCDIR}/meta-anthias-hybris \
-  ${SRCDIR}/meta-sparrow-hybris \
-  ${SRCDIR}/meta-sprat-hybris \
-  ${SRCDIR}/meta-tetra-hybris \
-  ${SRCDIR}/meta-bass-hybris \
-  ${SRCDIR}/meta-catfish-hybris \
-  ${SRCDIR}/meta-dory-hybris \
-  ${SRCDIR}/meta-lenok-hybris \
-  ${SRCDIR}/meta-ray-hybris \
-  ${SRCDIR}/meta-sturgeon-hybris \
-  ${SRCDIR}/meta-sawfish-hybris \
-  ${SRCDIR}/meta-skipjack-hybris \
-  ${SRCDIR}/meta-smelt-hybris \
-  ${SRCDIR}/meta-mtk6580-hybris \
-  ${SRCDIR}/meta-mooneye-hybris \
-  ${SRCDIR}/meta-narwhal-hybris \
-  ${SRCDIR}/meta-swift-hybris \
-  ${SRCDIR}/meta-wren-hybris \
-  "' > build/conf/bblayers.conf
+  ${SRCDIR}/meta-openembedded/meta-filesystems \' > build/conf/bblayers.conf
+
+        # Parse all to an array, find layers in meta-smartwatch, exclude root directory, remove most src/, sort alphabetically.
```

Comment:
The wording is a bit weird. I would reword as:

# Find all layers under src/meta-asteroid, remove the src/ prefix, sorted alphabetically, and store it in an array